content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Control Plane
+
+The Kubernetes control plane is the Kubernetes components such as the Kubernetes API Server, Kubernetes Controller Manager, and other components that are required for Kubernetes to funciton. Scalability limits of these components are different depending on what you’re running in the cluster, but the areas with the biggest impact to scale include the Kubernetes version, utilization, and individual Node scaling.
+
+## Use EKS 1.24 or above
+
+EKS 1.24 introduced a number of changes and switches the default container runtime to [containerd](https://containerd.io/) instead of docker. Containerd helps clusters scale by increasing individual node performance by limiting container runtime features to closely align with Kubernetes’ needs. Containerd is available in every supported version of EKS and if you would like to switch to containerd in versions prior to 1.24 please read the documentation about [Dockershim deprecation](https://docs.aws.amazon.com/eks/latest/userguide/dockershim-deprecation.html).
```

Comment:
Does that link actually tell you how to run containerd in versions prior to 1.24? If I'm not mistaken you had to modify userdata to use containerd. Let's make sure we have the right link here.

Reply:
Added a more appropriate link and re-worded a bit.